dc-cudami-editor/src/components/modals/imageAdder/ImageUploadForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react'
+import {Button, Form} from 'reactstrap'
+import {withTranslation} from 'react-i18next'
+
+import FileUploadForm from '../../FileUploadForm'
+import {uploadFile} from '../../../api'
+
+class ImageUploadForm extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      file: {},
+      progress: 0,
+      submitEnabled: false,
+    }
+  }
+
+  updateProgress = progress => {
+    this.setState({
+      progress,
+    })
+  }
+
+  render() {
+    const {t} = this.props
+    return (
+      <Form
+        onSubmit={async evt => {
+          evt.preventDefault()
+          const response = await uploadFile(
+            this.props.apiContextPath,
+            this.state.file,
+            'fileResource',
+            this.updateProgress
+          )
+          const responseJson = JSON.parse(response)
+          const label = Object.entries(responseJson.label)[0][1]
```

Comment:
Nachdem du nur den ersten Value willst ist `Object.values(responseJson.label)[0]` glaub besser.

```suggestion
          const label = Object.values(responseJson.label)[0]
```